### PR TITLE
Add grafana dashboard linter

### PR DESCRIPTION
**Pull request by arajkumar**
This commit introduces a linter[1] to verify grafana dashboard including
the promql expressions used in the dashboard.

[1] https://github.com/grafana/dashboard-linter

Fixes https://github.com/kubernetes-monitoring/kubernetes-mixin/issues/713

Signed-off-by: Arunprasad Rajkumar <arajkuma@redhat.com>

### CODE REVIEW

File: .lint [hunk 1]
```diff
@@ -0,0 +1,3 @@
+exclusions:
+ template-job-rule:
+ panel-job-instance-rule:
```
Comment (paulfantom) [inline .lint:3]: Can we have a follow-up issue to remove those exclusions?

**Review by paulfantom — APPROVED** [review]
Awesome work! :100: 

lgtm

File: .lint (hunk 1 above)
Comment (paulfantom) [reply .lint:3]: You are right, having `job` and `instance` won't be possible right now for all dashboards.

**Comment by arajkumar** [comment]
Thanks @paulfantom

**Comment by chrism417** [comment]
This broke the directions, specifically 'make dashboards_out' because it was removed in this commit.

**Pull request merged**